doc/tutorials/using_windows_as_uos.rst

```diff
@@ -72,43 +72,82 @@ Create a raw disk
 Run these commands on the Service VM::
 
    $ sudo apt-get install qemu-utils
-   $ mkdir /home/acrn/img
-   $ cd /home/acrn/img
+   $ mkdir /home/acrn/work
+   $ cd /home/acrn/work
    $ qemu-img create -f raw win10-ltsc.img 30G
 
 Prepare the script to create an image
 -------------------------------------
 
-#. Copy ``/usr/share/acrn/samples/nuc/launch_win.sh`` to ``install_win.sh``::
-
-   $ cp /usr/share/acrn/samples/nuc/launch_win.sh /home/acrn/img/install_win.sh
-
-
-#. Edit the ``acrn-dm`` command line in ``install_win.sh`` as follows:
-
-   .. note:: Make sure you use GVT-g ``-s 2,pci-gvt -G "$2"`` in the
-      ``acrn-dm`` command line. Currently, we do not support creating a
-      windows image by GVT-d.
-
-   - Change ``-s 3,virtio-blk,./win10-ltsc.img`` to your path to the Windows
-     10 image.
+#. Refer to :ref:`gpu-passthrough` to enable GVT-d GOP feature. Then
+   copy above ``.iso`` files and the build ``OVMF.fd`` to
+   ``/home/acrn/work``
+#. Prepare WaaG install script
+
+.. code-block:: bash
+
+   vim /home/acrn/work/install_win.sh
```

Comment:
Probably best to split this in a different code-block, or just mention in the section title that a `/home/acrn/work/install_win.sh` file should be created with the following content.

Reply:
Done!